build_metal.sh

```diff
@@ -1,4 +1,50 @@
 #!/bin/bash
+: '
+TLDR: Follow the steps outlined below to build metal. For more in-depth information keep reading
+
+Steps:
+    1. Configure and generate build files: `cmake -B build -G Ninja`
+        - the `-B` indicates where the build folder is, you can change the folder name to whatever you want
+        - the `-G Ninja` specifies to cmake to use the Ninja build system, which is faster and more reliable than make
+    2. Build metal: `ninja install -C build`
+        - the general command would be: `cmake --build build --target install`
+        - the `-C` indicates where to run the command, in this case it will be your build folder(s)
+        - we are targeting `install` since that will also just build src
+        - the install target will install pybinding .so (_C.so & _ttnn.so) into the src files, so pybinds can be used
+    3. Build cpp tests: `ninja tests -C build`
+        - building tests will also automatically build src
+    4. Create python_env (you only need to do this ONCE): ./scripts/build_scripts/create_venv.sh
+        - the env will default get created in $TT_METAL_HOME/python_env, if you want to change it set the PYTHON_ENV_DIR=
+        - this step is not dependent on any of the other steps, you only need to run ONCE and in any order
+
+Notes:
+    - YOU ONLY NEED TO BUILD THE PYTHON_ENV ONCE!!!!! (unless you touch the dev python dependencies)
```

Comment:
I'm leaning towards stripping the `create_venv.sh` out of `build_metal.sh` altogether and have the user do:

1) `create_venv.sh`
2) source python_env/bin/activate
3) `build_metal.sh`

Reply:
create_venv has been removed